app/database/permission_granted_store_compute_all_access.go

```diff
@@ -34,14 +34,14 @@ func (s *PermissionGrantedStore) computeAllAccess() {
 		JOIN permissions_generated AS parents
 			ON parents.item_id = items_items.parent_item_id
 		JOIN permissions_propagate AS parents_propagate
-			ON parents_propagate.group_id = parents.group_id AND parents_propagate.item_id = parents.item_id AND
-				 parents_propagate.propagate_to = 'children'
```

Comment:
how do you find this kind of improvements?

Reply:
I usually go through different alternatives with EXPLAIN and EXPLAIN ANALYZE